docs/textbook/nbinteract_Graphing.ipynb

```diff
@@ -288,6 +240,69 @@
     "nbinteract.scatter_drag(x_coords, y_coords, options={'xlim': (0, 9), 'ylim': (0, 11)})"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### nbinteract.bar"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The first two arguments of `nbinteract.bar` are functions to generate the x and y-axis data, respectively. Either argument can be arrays themselves. Any argument used in those functions must be passed in. The function generates an interactive bar plot where the parameters for tose functions can be interacted with. "
```

Comment:
nit: can we call the functions `response functions`? That way we can disambiguate between the `nbinteract.bar` function and its response functions.

I know I wrote this description lol but maybe the description is a bit more clear worded like:

> The first two arguments of `nbinteract.bar` are response functions that return the x and y-axis data arrays, respectively. Either argument can be arrays themselves. Each argument for the response functions must be passed in as keyword arguments to `nbinteract.bar` in the format expected by `interact`. The response function for the y-axis data will be called with the x-axis data as its first argument.
> 
> `nbinteract.bar` generates an bar plot that allows interaction with the parameters for the response functions.

I think this avoid the passive voice and more clearly distinguishes between the function and its response functions. But feel free to change the wording above when you push.